libs/dal/src/repositories/notification-template/notification-template.entity.ts

```diff
@@ -142,11 +141,11 @@ export class StepVariantEntity implements IStepVariant {
   shouldStopOnFail?: boolean;
 
   bridgeUrl?: string;
-  /**
-   * @deprecated This property is deprecated and will be removed in future versions.
-   * Use `fullName` instead.
+  /*
+   * controlVariables exists
+   * only on none production environment in order to provide stateless control variables on fly
    */
-  controlVariables?: ControlsDto;
+  controlVariables?: Record<string, unknown>;
```

Comment:
Is this property deprecated today or not yet?

Reply:
it was a misuse that worked just because there is no real types in javascript, this was not supposed to be the type here. verified in details pairing with @djabarovgeorge